modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/util/Commons.java

```diff
@@ -896,4 +899,11 @@ public static List<RelNode> castInputsToLeastRestrictiveTypeIfNeeded(List<RelNod
 
         return actualInputs;
     }
+
+    /** Returns {@code true} if the specified properties allow multi-statement query execution. */
+    public static boolean isMultiStatementQuery(SqlProperties properties) {
```

Comment:
maybe it is better to rename it to reflect the real purpose of the method, e.g. `isMultiStatementQueryAllow'